src/java/generator/MapGenerator.java

```diff
@@ -333,67 +359,67 @@ private static void interpretArguments(Map<String, String> arguments) {
 			System.exit(0);
 		}
 
-		if (!Arrays.asList("folder-path").stream().allMatch(arguments::containsKey) && !DEBUG) {
+		if (!Stream.of("folder-path").allMatch(arguments::containsKey) && !DEBUG) {
 			System.out.println("Missing necessary argument.");
 			System.exit(-1);
 		}
 
-		FOLDER_PATH = arguments.get("folder-path");
+		folderPath = arguments.get("folder-path");
 
 		if (arguments.containsKey("map-name")) {
-			MAP_NAME = arguments.get("map-name");
+			mapName = arguments.get("map-name");
 			parseMapName();
 			return;
 		}
 
 		if (arguments.containsKey("seed")) {
-			SEED = Long.parseLong(arguments.get("seed"));
-			RANDOM = new Random(SEED);
+			seed = Long.parseLong(arguments.get("seed"));
+			random = new Random(seed);
 		}
 		randomizeOptions();
 
 		if (arguments.containsKey("spawn-count")) {
-			SPAWN_COUNT = Integer.parseInt(arguments.get("spawn-count"));
+			spawnCount = Integer.parseInt(arguments.get("spawn-count"));
 		}
 
 		if (arguments.containsKey("land-density")) {
-			LAND_DENSITY = Float.parseFloat(arguments.get("land-density"));
-			LAND_DENSITY = (float) StrictMath.round(LAND_DENSITY*127)/127;
+			landDensity = Float.parseFloat(arguments.get("land-density"));
+			landDensity = (float) StrictMath.round(landDensity *127)/127;
 		}
 
 		if (arguments.containsKey("plateau-density")) {
-			PLATEAU_DENSITY = StrictMath.min(Float.parseFloat(arguments.get("plateau-density")),0.2f);
-			PLATEAU_DENSITY = (float) StrictMath.round(PLATEAU_DENSITY*127)/127;
+			plateauDensity = StrictMath.min(Float.parseFloat(arguments.get("plateau-density")),0.2f);
+			plateauDensity = (float) StrictMath.round(plateauDensity *127)/127;
 		}
 
 		if (arguments.containsKey("mountain-density")) {
-			MOUNTAIN_DENSITY = StrictMath.min(Float.parseFloat(arguments.get("mountain-density")),0.1f);
-			MOUNTAIN_DENSITY = (float) StrictMath.round(MOUNTAIN_DENSITY*127)/127;
+			mountainDensity = StrictMath.min(Float.parseFloat(arguments.get("mountain-density")),0.1f);
+			mountainDensity = (float) StrictMath.round(mountainDensity *127)/127;
 		}
 
 		if (arguments.containsKey("ramp-density")) {
-			RAMP_DENSITY = StrictMath.min(Float.parseFloat(arguments.get("ramp-density")),0.2f);
-			RAMP_DENSITY = (float) StrictMath.round(RAMP_DENSITY*127)/127;
+			rampDensity = StrictMath.min(Float.parseFloat(arguments.get("ramp-density")),0.2f);
+			rampDensity = (float) StrictMath.round(rampDensity *127)/127;
 		}
 
 		if (arguments.containsKey("reclaim-density")) {
-			RECLAIM_DENSITY = Float.parseFloat(arguments.get("reclaim-density"));
-			RECLAIM_DENSITY = (float) StrictMath.round(RECLAIM_DENSITY*127)/127;
+			reclaimDensity = Float.parseFloat(arguments.get("reclaim-density"));
+			reclaimDensity = (float) StrictMath.round(reclaimDensity *127)/127;
```

Comment:
`127f` and spacing for binary operators and after commas.

Reply:
Done